spec/support/mockPartnerConfig.json

```diff
@@ -1,27 +1,27 @@
 {
-    "timeout": 1000,
+    "timeout": 3000,
     "xSlots": {
         "1": {
-            "placementId": "54321",
+            "inScreen": "ggumtest",
             "sizes": [
```

Comment:
For clarification purposes, assuming a publisher defined the following dfp slot A with size 300x250 and 300x600, would there be one xSlot `1 with sizes: [[300, 250], [300, 600]`, or two xSlots `1:{size: [[300,250]]}`and `2: {size: [[300,600]]}`?

Reply:
In the current config as shown in the file we're assuming 2 configured DFP slots.

So in the case there's one DFP Slot A [[300x250],[300x600]] I think that, for us would be a single xSlot.